### PR TITLE
Skip deezer empty image

### PR DESCRIPTION
**Pull request by soncaokim**
this is a proposal to avoid pulling deezer's place holder image in artist view.

the detection code is somewhat fragile, any idea to improve is welcomed.

### CODE REVIEW

File: app/src/main/java/com/poupa/vinylmusicplayer/glide/artistimage/ArtistImageFetcher.java [hunk 1]
```diff
@@ -68,22 +71,36 @@ public void onResponse(@NonNull Call<DeezerResponse> call, @NonNull Response<Dee
...
-                            streamFetcher = new OkHttpStreamFetcher(okhttp, new GlideUrl(url));
-                            streamFetcher.loadData(priority, callback);
+                            List<Data> data = deezerResponse.getData();
+                            if (data.size() > 0) {
+                                String url = data.get(0).getPictureMedium();
+
+                                // Fragile way to detect a place holder image returned from Deezer:
+                                // ex: "https://e-cdns-images.dzcdn.net/images/artist//250x250-000000-80-0-0.jpg"
+                                // the double slash implies no artist identified
+                                final boolean placeholderUrl = url.contains("/images/artist//");
```
Comment (AdrienPoupa) [inline app/src/main/java/com/poupa/vinylmusicplayer/glide/artistimage/ArtistImageFetcher.java]: maybe we can simply detect if there are more than one occurrence of `//`?

Comment (soncaokim) [reply app/src/main/java/com/poupa/vinylmusicplayer/glide/artistimage/ArtistImageFetcher.java]: @paolovalerdi Do you have any comment on this, any better way to detect the place holder image? (Seems that I cannot add you a a reviewer).
@AdrienPoupa that would certainly work, but the detection remains fragile.

**Comment by soncaokim** [comment]
Hi, I've cleanup the code to remove all experiments/debug.
@AdrienPoupa please have a look, I dont think I can make it really better.

**Comment by AdrienPoupa** [comment]
Looks good to me thanks!

**Pull request merged**